solutions/gold/usaco-862.mdx

```diff
@@ -0,0 +1,105 @@
+---
+id: usaco-862
+source: USACO Gold 2018 December
+title: Cowpatibility
+author: Ryan Chou
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_cowpatibility_gold_dec18.html)
+
+<Spoiler title="Hint">
+
+Instead of trying to find the number of incompatible pairs, try to find the number of *compatible* pairs instead!
+
+This is known as [complementary counting](https://artofproblemsolving.com/wiki/index.php/Complementary_counting).
+
+</Spoiler>
+
+<Spoiler title="Solution">
+
+## Explanation
+
+Since there are $\frac{N(N-1)}{2}$ total pairs of cows, if we can count the numbers of compatible pairs, we can just subtract it from the total to get our answer!
```

Comment:
```suggestion
Since there are a total of $\frac{N(N-1)}{2}$ pairs of cows, if we can count the number of compatible pairs, we can subtract it from the total to get our answer!
```